docs/Local_Development_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+- How to debug the Kubernetes environment 
+- How to clean up your environment
+
+
+## Prerequisites
+The following steps assume you are running this tutorial from a desktop environment. Mobile environments will likely experience problems and may not work.
+- [Docker](https://docs.docker.com/engine/install/): Subscribe to Docker Hub and pulldown the latest image from: https://hub.docker.com/_/marklogic
+  ```sh
+  # Something similar to this, with the latest version tag, which can be found on the dockerhub link above
+  docker pull store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4 
+  ```
+- [KubeCTL](https://kubernetes.io/docs/tasks/tools/):  Download and install this tool to assist with debugging in a Kubernetes environment. 
+- [Helm](https://helm.sh/docs/intro/install/):  Clone or download the chart repository: https://github.com/marklogic/marklogic-kubernetes
+- [Minikube](https://k8s-docs.netlify.app/en/docs/tasks/tools/install-minikube/): Minikube Kubernetes environment, which will host the MarkLogic Server application.
+- Mac OSX VM: If you are running in Mac OSX, you will need Virtual Box: [Virtual Box](https://www.virtualbox.org/) the VM used inside of Minikube. 
+- Browser: The latest version of a supported web browser. See the list here: [Web Browser](https://developer.marklogic.com/products/support-matrix/) 
+
+
+# Procedure 
+This section describes the procedure for setting up Minikube, installing MarkLogic Server, and verifying the installation. 
+
+
+## Setting Up Minikube
+First you will need to set up the Kubernetes control plane on your local machine. Minikube is a tool that makes it easy to set up a local Kubernetes enviornment.
+
+- Start Minikube: `minikube start --driver=virtualbox`
+ If you are running in a Linux OS: `minikube start --driver=docker`  
+
+To verify the Minikube started correctly, use the Kubernetes command line tool, KubeCTL:  
+
+```sh
+kubectl get nodes
+```
+```
+Expected Output: 
+NAME       STATUS   ROLES                  AGE   VERSION
+minikube   Ready    control-plane,master   1d    v1.23.3
+```
+
+ To enable addons run the follow Minikube command for ingress:     
+ `minikube addons enable ingress`  
+##  Installing a Single MarkLogic Host to Minikube
+- Push the image used for MarkLogic Server to the VM:   
+`minikube image load store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4`  
+  The image ID used in the example is `store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4`, which may not be the latest image. To find the latest ID go to https://hub.docker.com/_/marklogic  
+- Navigate to where you downloaded or cloned the MarkLogic Helm repository.
+  Verify that the image loaded to Minikube with the comand above matches the `repository` and `tag` in the `values.yaml`.
+  ```YAML
+  image:
+    repository: store/marklogicdb/marklogic-server
+    tag: 10.0-9-centos-1.0.0-ea4
+  ```
+  Navigate to the `/charts` folder in your local directory. 
```

Comment:
Since we might not want the user to use source code to install chart directly, they can provide their own values file when they install marklogic chart.

Reply:
Ok i'll provide a reference link to our `values.yaml` file for them to use as well.

Reply:
0c45940 - Values.yaml inclusion